### ISSUE TITLE
Better Background Image 

### ISSUE THREAD

**Issue by DhairyaBahl** [DhairyaBahl/React-Messenger-App#74]
### Description 

Currently the app is using whatsapp background images In light mode and also in the dark mode . Suggestions Open for background image for this App.

**Comment by AnjaliPatle** [comment]
What do you think about using similar kind of patterns for background? I have attached an example below.

![image](https://user-images.githubusercontent.com/50516607/111026178-077de880-840f-11eb-8147-e015b28667da.png)

![image](https://user-images.githubusercontent.com/50516607/111026165-f59c4580-840e-11eb-9a8b-30522f62924d.png)

**Comment by DhairyaBahl** [comment]
@AnjaliPatle Images are pretty good but I think that it becomes a bit distracting in light mode ! what do you say ? Still I am waiting for few more suggestions.

**Comment by SanjayDevTech** [comment]
Can u assign this to me?

**Comment by DhairyaBahl** [comment]
@SanjayDevTech Feel free to  give your suggestions but due to some reasons this issue can't be assigned to anyone. Although if your suggestion is chosen then this issue will be definitely assigned to you.

**Comment by yokesh004** [comment]
> ### Description
> Currently the app is using whatsapp background images In light mode and also in the dark mode . Suggestions Open for background image for this App.

What about these linear gradients:
for light mode:
1.
![image](https://user-images.githubusercontent.com/59578892/111779066-aad17080-88db-11eb-99cd-1ea85f2d7247.png)

(or)

2.
![image](https://user-images.githubusercontent.com/59578892/111779636-609cbf00-88dc-11eb-9a05-3a70bf723c94.png)

For dark mode:
1.
![image](https://user-images.githubusercontent.com/59578892/111779120-ba50b980-88db-11eb-9fef-46b496ae09f6.png)

(or)

2.
![image](https://user-images.githubusercontent.com/59578892/111779428-22070480-88dc-11eb-9836-086bc03901b3.png)

 (or)

3.
![image](https://user-images.githubusercontent.com/59578892/111779862-a6598780-88dc-11eb-8535-ba6f3976d43f.png)

**Comment by DhairyaBahl** [comment]
@yokesh004 The last one is good ! But everything looks very empty ! It would be great if there is some design in the background !

**Comment by riya3000** [comment]
How about sunrise/day sky for light mode
1 ![fuyu-yeo-jN329Z_wEYQ-unsplash](https://user-images.githubusercontent.com/72564975/112663760-32405600-8e7f-11eb-8131-d8c8accf605c.jpg)
or 2 ![s-o-c-i-a-l-c-u-t-pIwrG4NhxXI-unsplash](https://user-images.githubusercontent.com/72564975/112664111-9b27ce00-8e7f-11eb-957f-3add5deca3dc.jpg)

and sunset/night sky for dark mode
1 ![oli-Cw3mOeWufOA-unsplash](https://user-images.githubusercontent.com/72564975/112664271-cad6d600-8e7f-11eb-93c1-7c59b3329c49.jpg)
or 2 ![jari-hytonen-VJBIn6n_gzI-unsplash](https://user-images.githubusercontent.com/72564975/112664862-72ec9f00-8e80-11eb-9962-580b102c06ab.jpg)
If you like these background images, please assign me here I am a gssoc'21 participant. Thank u @DhairyaBahl

**Comment by Ashishkubehera** [comment]
![sunset-404072](https://user-images.githubusercontent.com/74497927/113274908-d43ac500-92fb-11eb-8a30-2aac8640f82b.jpg)
If you like this image then please assign me the issue.
I'm a gssoc participant.

**Comment by Ashishkubehera** [comment]
![tropical-1651423_1920](https://user-images.githubusercontent.com/74497927/113276157-2e885580-92fd-11eb-9ea3-339d66807ab6.jpg)

**Comment by DhairyaBahl** [comment]
@riya3000 Would you still like to work on this issue ? I am asking because you commented first ? Tell me , otherwise I will assign this issue to @Ashishkubehera .

**Comment by DhairyaBahl** [comment]
Sorry for delay from my side ! I am bit busy these days !

**Comment by riya3000** [comment]
Yes, of course I still want to work on this issue. Let me know which of the above 2 images you like for both light and dark mode. Please assign me to this issue Thank you @DhairyaBahl

**Comment by DhairyaBahl** [comment]
@riya3000 I am assigning this issue to you. Send the screenshots of all the images as example in background. That way I will be able to choose more easily that which one suits the best to the app.

@Ashishkubehera Sorry, for late reply, do check out other issues. If you can't find any issues, create one that you want an issue to work on and tag me and other collaborators as well and we will get you started. 

All the best ^_^

**Comment by riya3000** [comment]
I have uploaded the screenshot of all images and some new images on this link. Please take a look
[Background Themes](https://www.figma.com/file/s11WEczRgegUwbA4qp3vLO/Background-img?node-id=0%3A1)
Let me know which one you like or do you want me to look up some more images. Thank you @DhairyaBahl

**Comment by DhairyaBahl** [comment]
@riya3000  I've seen all the images ! first of all, really appreciate the effort ! Well Done and thank you !

Secondly , first 4 images are designs, we should use monochromatic shades like in the rest 4 which will make the app look more modern and stylish. 

Out of the rest 4 , all of them seems to be good to me ! So, how about we create a menu for the same, and in that menu, user can choose which one he wants to use as background. Later we can also add the functionality to allow user to add his own images as background. Hope you got my point. but that will be done once authentication feature is added.

**Comment by riya3000** [comment]
Ok I get your point. Creating a drop down menu bar with options like background image, font style, font size, and other things will be a good idea. I will definitely work on it. Then what should I do for this issue, should I make a pr changing the background img to 
![Messenger - Google Chrome 03-04-2021 11_34_49 (2)](https://user-images.githubusercontent.com/72564975/113473763-370d9700-9489-11eb-8e46-2b8f8a3ca048.png)
for light mode 
![Messenger - Google Chrome 03-04-2021 11_35_02 (2)](https://user-images.githubusercontent.com/72564975/113473769-44c31c80-9489-11eb-8747-a6aa0505bcb7.png)
and this for dark mode as you said current img for light mode is a bit distracting. Then you can create a new issue for the menu bar and I can work on it. Thank You @DhairyaBahl

**Comment by DhairyaBahl** [comment]
@riya3000 What I am suggesting is, this issue will be a sub issue for the menu bar issue then you can work on the whole menubar which will solve >2 issues because each and every feature will be counted as one issue. So that would be my approach. 

But for the time being, you can set the background image of light mode to your suggested one and create a pr for the same. Don't make any changes in the dark mode because I think that we can find better image for the same.

**Comment by subhangi2731** [comment]
@abhijeet007rocks8 please assign this issue to me

**Comment by Divyanshj** [comment]
Is this issue still open ?
If yes then i want to show my suggestions for the background image.
please confirm
